app/helpers/record_helper.rb

```diff
@@ -118,6 +118,13 @@ def source_metadata_link(links)
     links.select { |link| link['kind'] == 'Download' && link['text'] == 'Source Metadata' }.first['url']
   end
 
+  # Geopoint is deprecated in the API, but that seems to be where all geospatial data is mapped. Including both for now.
```

Comment:
This doesn't seem good. Are we sure this isn't just because TIMDEX UI is requesting the wrong field? TIMDEX API maps the data to both places but maybe we are requesting it from the wrong one in our query?

Reply:
```
{
  search(searchterm: "rail", index: "geo") {
    hits
    records {
      title
      locations {
        geoshape
        geopoint
      }
    }
  }
}
```

Returns data in both fields (with geopoints being deprecated). I suspect what you are seeing is a TIMDEX UI oddity unless you have examples where it is working differently than I am seeing/understanding.

Reply:
Gah! You're right. 🤦 I thought I'd cross-checked this in the API, but it's duplicated in both fields as expected.

Reply:
Yeah I think it's this that needs updating:
https://github.com/MITLibraries/timdex-ui/blob/main/app/models/timdex_record.rb#L63

I suspect we missed the "recordID" method updating when we added geo searches as we likely were focused only on searching and not full record views with that work.

Reply:
Do you want to roll that into this PR or do a separate one for that and land it before this? I'm good with either.

Reply:
I'd like to roll it into this PR, but I'm having trouble getting the tests to not error with the regenerated cassettes. Stay tuned...

Reply:
I must have just hit a run of bad luck. The test suite started passing after I ran it a handful of times. I'm glad that erratic behavior seems to mostly be limited to our locals...